src/components/DownloadContactListButton.tsx

```diff
@@ -15,8 +15,12 @@ export function DownloadContactListButton (props: DownloadButtonProps) {
     const filteredPropObject = getUniqueListBy(arrayOfPropertyObjects, 'id')
     filteredPropObject.forEach(result => {
         if (typeof result.owner_details !== 'undefined' && Array.isArray(result.owner_details)) {
-            const mobile = result.owner_details[0].Mobile || result.owner_details[1]?.Mobile
-            const workPhone = result.owner_details[0].Work_Phone || result.owner_details[1]?.Mobile
+            let mobile
+            let workPhone
+            result.owner_details.map((owner) => {
+                mobile = owner.Mobile
```

Comment:
this won't solve the problem because owner.Mobile could be null. array.prototype.map produces an array. If you want to do it this way, you'd need to use forEach and do an existence check, e.g.

```owner.Mobile && mobile = owner.Mobile;```